sdk/src/Services/S3Control/Generated/Model/Internal/MarshallTransformations/CreateJobRequestMarshaller.cs

```diff
@@ -288,8 +288,8 @@ public IRequest Marshall(CreateJobRequest publicRequest)
                                 xmlWriter.WriteElementString("key", kvp.Key);
                                 xmlWriter.WriteElementString("value", kvp.Value);
                                 xmlWriter.WriteEndElement();
-                            }            
-                            xmlWriter.WriteEndElement();                
```

Comment:
Sorry in advance for these whitespace diffs, I don't know how they got there and i couldn't remove them

Reply:
Yes, it was one I missed from this commit which we PRed: https://github.com/aws/aws-sdk-net/pull/3243/commits/b19a0105bfa967cbd04f57d38709c6c699db1aff